modules/rhel-adding-node.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+
+[id="rhel-adding-node-{context}"]
+= Adding a RHEL compute machine to your cluster
+
+You can add compute machines that use Red Hat Enterprise Linux as the operating
+system to a {product-title} {product-version} cluster.
+
+.Prerequisites
```

Comment:
This could be updated to be `Control Host Prerequisites`.  We are missing the requirements of installing Ansible 2.7 and installing the openshfit-ansible rpm.  The item about preparing RHEL hosts seems to not apply if we use this as Control Host Prereqs.

Reply:
`.Prerequisites` is the standard section header for all of the procedure "modules" that we're using in 4.1, and the item about preparing RHEL hosts is meant to tell users that they have to complete that section first.

Reply:
Ok, so we need to specify more details on where to install or configure things.

Reply:
@mtnbikenc, I made changes around the prereqs and added a separate module about configuring the host that you run the playbook on. Will you please take another look?